### PR TITLE
Read non well-known keys from a config file

### LINKED ISSUE #37: Add ability to read non well-known keys from a config file
When we deploy Faucet on the real TestNet, the keys used will not be well-known.
Hence, Faucet should be able to read a config file and store the keys it knows about.

### PR DESCRIPTION
**Pull request by kchulj**
Faucet can read a config file and store the keys in the Config struct.

Fixes #37

### CODE REVIEW

File: dub.json [hunk 1]
```diff
@@ -43,11 +43,13 @@
         "submodules/agora/source/agora/utils/Test.d",
         "submodules/agora/source/agora/utils/TxBuilder.d",
         "submodules/agora/source/agora/utils/Utility.d",
-        "submodules/agora/source/agora/utils/WellKnownKeys.d"
+        "submodules/agora/source/agora/utils/WellKnownKeys.d",
+        "submodules/dyaml/source/dyaml/loader.d"
```
Comment (Geod24) [inline dub.json:47]: This is not needed. We only use that hack in Agora because otherwise we pull in dependency to SCP.

Comment (kchulj) [reply dub.json:47]: Ah is this why there are errors in the CI?

**Comment by Geod24** [comment]
Yep

File: dub.json (hunk 1 above)
Comment (Geod24) [reply dub.json:47]: Ye[

**Review by Geod24 — APPROVED** [review]
You parse it, now we have to use it

**Comment by kchulj** [comment]
🟢

**Pull request merged**